Week2/homework/maartjes-work.js

```diff
@@ -46,16 +46,20 @@ const maartjesTasks = monday.concat(tuesday);
 const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
-  // Replace this comment and the next line with your code
-  console.log(tasks, hourlyRate);
+  const taskHours = tasks.map(x => x.duration / 60);
```

Comment:
Variables with name `x` should not be used unless it represents an x-coordinate. Please use meaningful names as variable names, also in callback and arrow functions.